src/js/script.js

```diff
@@ -3,96 +3,78 @@ const titleInput = document.getElementById('title-input');
 const authorInput = document.getElementById('author-input');
 const booksList = document.getElementById('books');
 
-
 let books = [];
 let id = 0;
 
-// Creating the book and getting user input
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
+
+class UI {
+  static loadBooks() {
+    const storedBooks = JSON.parse(localStorage.getItem('books'));
```

Comment:
- [ ] Nice work @team, however, there are 2 items that are present anytime before and after you add or remove a book, please make sure you fix this and will be good to go
<img width="673" alt="1" src="https://github.com/Rebelzob/Awesome-books/assets/77843050/e11ed256-a5fb-4ef9-b8da-d0916e059bd0">

Reply:
Hello @joskalenda here is a video on how it is working for us in our browsers 

https://www.loom.com/share/59872e80ea3b4b35abf09c26da86bbf6